prog/weaver/metrics.go

```diff
@@ -63,15 +61,19 @@ var metrics []metric = []metric{
 	{desc("weave_ips", "Number of IP addresses.", "state"),
 		func(s WeaveStatus, desc *prometheus.Desc, ch chan<- prometheus.Metric) {
 			if s.IPAM != nil {
-				ch <- intGauge(desc, s.IPAM.RangeNumIPs, "total")
 				ch <- intGauge(desc, s.IPAM.ActiveIPs, "local-used")
 			}
 		}},
-	{desc("weave_dns_entries", "Number of DNS entries.", "state"),
+	{desc("weave_max_ips", "Number of IP addresses used by allocator."),
```

Comment:
I read this description as "# of allocated IPs". Maybe "the size of IP address space" is more accurate?